legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/gitlab/api/GitLabReviewApi.java

```diff
@@ -644,6 +661,47 @@ public ReviewUpdateStatus updateReview(String projectId, String reviewId)
         return getReviewUpdateStatus(gitLabProjectId, gitLabApi, rebaseMergeRequest);
     }
 
+    @Override
+    public Review editReview(String projectId, String reviewId, String title, String description, List<String> labels)
+    {
+        LegendSDLCServerException.validateNonNull(projectId, "projectId may not be null");
+        LegendSDLCServerException.validateNonNull(reviewId, "reviewId may not be null");
+        LegendSDLCServerException.validateNonNull(title, "title may not be null");
+        LegendSDLCServerException.validateNonNull(description, "description may not be null");
+
+        GitLabProjectId gitLabProjectId = parseProjectId(projectId);
+        GitLabApi gitLabApi = getGitLabApi(gitLabProjectId.getGitLabMode());
+        MergeRequestApi mergeRequestApi = gitLabApi.getMergeRequestApi();
+
+        MergeRequest mergeRequest = getReviewMergeRequest(mergeRequestApi, gitLabProjectId, reviewId);
+        if (!isOpen(mergeRequest))
+        {
+            throw new LegendSDLCServerException("Only open reviews can be edited: state of review " + mergeRequest.getIid() + " in project " + gitLabProjectId.toString() + " is " + getReviewState(mergeRequest));
+        }
+        try
+        {
+            MergeRequestParams mergeRequestParams  = new MergeRequestParams().withTitle(title).withDescription(description);
+            if (labels != null && !labels.isEmpty())
+            {
+                mergeRequestParams.withLabels(labels);
+            }
```

Comment:
What if someone wants to remove all labels? Perhaps we should distinguish between labels == null (meaning that there should be no change in the labels) and labels.isEmpty() (meaning that all labels should be removed).